client/blocks/app-banner/docs/example.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import AppBanner from '../';
+import { PREFERENCE_NAME } from '../utils';
+import { savePreference } from 'state/preferences/actions';
+
+function AppBannerExample( clearPreference ) {
```

Comment:
Removed in 0231c21a5cc3173a627ef787cd2ac08a24712505 for now in order to focus on new PR feedback.